pkgs/os-specific/linux/atop/atopacct.service.patch

```diff
@@ -0,0 +1,7 @@
+--- a/atopacct.service
++++ b/atopacct.service
+@@ -9,3 +9,3 @@
+ Type=forking
+-PIDFile=/var/run/atopacctd.pid
++PIDFile=/run/atopacctd.pid
```

Comment:
Those should be symlinked.

Reply:
see https://github.com/NixOS/nixpkgs/pull/123053#discussion_r633151414